Lib/multiprocessing/context.py

```diff
@@ -5,7 +5,7 @@
 from . import process
 from . import reduction
 
-__all__ = []            # things are copied from here to __init__.py
+__all__ = []
```

Comment:
Could we use `__all__ = ()`?  Or `__all__` is modified later?

Reply:
I will run the test and tell you the result

Reply:
@methane  I updated things that you pointed out. I passed all tests. Thanks.